recipe/meta.yaml

```diff
@@ -64,17 +67,19 @@ test:
     - jupyter nbconvert nbconvert/tests/files/notebook1.ipynb --to html
 
 about:
-  home: https://jupyter.org
+  home: https://nbconvert.readthedocs.io/en/latest/
   license: BSD-3-Clause
   license_file: LICENSE
   license_family: BSD
+  license_url: https://github.com/jupyter/nbconvert/blob/6.5.3/LICENSE
```

Comment:
there is a license_file in package.  No need for the license_url.  It is superflous, even problematic mid/long term

Reply:
done for you ...